gnome-initial-setup/pages/summary/gis-summary-page.c

```diff
@@ -234,13 +234,6 @@ add_setup_done_file (void)
 static void
 done_cb (GtkButton *button, GisSummaryPage *page)
 {
-  gchar *file;
```

Comment:
@alepuccetti I am suggesting that such commit is reduced to simply remove/comment out that code, and not touching any other parts of e.g. the build system, since that should be enough not to trigger the GNOME welcome tour. Alternatively, we should also remove the `gnome-welcome-tour` file itself from the git repo.